### ISSUE TITLE
Multiple windows example broken after new states

### ISSUE THREAD

**Issue by jakobhellermann** [bevyengine/bevy#1709]
Since 284889c64b2a4a4f67018e42a14062392edbd0aa running the multiple windows example results in
```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', examples/window/multiple_windows.rs:75:10
```
https://github.com/bevyengine/bevy/blob/45b2db70705da24a89426e6b6e77d603a3983025/examples/window/multiple_windows.rs#L71-L75

**Comment by bjorn3** [comment]
https://github.com/bevyengine/bevy/blob/45b2db70705da24a89426e6b6e77d603a3983025/examples/window/multiple_windows.rs#L21-L26

`setup_pipeline` runs before `setup_window` I think.

**Comment by jakobhellermann** [comment]
The `SystemSet::on_enter(AppState::CreateWindow)` should be `SystemSet::on_enter(AppState::Setup)`.

But even then, the order is
1. run set_window
2. run setup_pipeline (windows.iter() is only the primary window)
3. run setup_pipeline (windows.iter() is only the primary window and the new one)

**Comment by cart** [comment]
Resolved by #1737
